### PR TITLE
[zh] Translate delete-options page into Chinese

### PR DESCRIPTION
**Pull request by twilight0620**
Fixes: https://github.com/kubernetes/website/issues/32186
#32186  

Translate:  https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/delete-options/  into Chinese

### CODE REVIEW

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+<!--
+- **propagationPolicy** (string)
+
+  Whether and how garbage collection will be performed. Either this field or OrphanDependents may be set, but not both. The default policy is decided by the existing finalizer set in the metadata.finalizers and the resource-specific default policy. Acceptable values are: 'Orphan' - orphan the dependents; 'Background' - allow the garbage collector to delete the dependents in the background; 'Foreground' - a cascading policy that deletes all dependents in the foreground.
+-->
+
+- **propagationPolicy** (string)
+
+  表示是否以及如何执行垃圾收集。可以设置此字段或 `OrphanDependents` 字段，但不能同时设置。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:138]:
```suggestion
  表示是否以及如何执行垃圾收集。可以设置此字段或 `orphanDependents` 字段，但不能同时设置二者。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  
+  默认策略由 `metadata.finalizers` 中设置的现有终结器（Finalizer）和特定资源的默认策略决定。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:140]:
```suggestion
  默认策略由 `metadata.finalizers` 中现有的终结器（Finalizer）集合和特定资源的默认策略决定。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  
+  可接受的值为： `Orphan` - 依赖对象是孤儿；`Background` - 允许垃圾收集器在后台删除依赖项；`Foreground` - 一个级联策略，删除前台中的所有依赖项。
```
Comment (tengqm) [inline content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:142]:
```suggestion
  可接受的值为： `Orphan` - 令依赖对象成为孤儿对象；`Background` - 允许垃圾收集器在后台删除依赖项；
  `Foreground` - 一个级联策略，前台删除所有依赖项。
```

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+api_metadata:
+  apiVersion: ""
+  import: "k8s.io/apimachinery/pkg/apis/meta/v1"
+  kind: "DeleteOptions"
+content_type: "api_reference"
+description: "删除 API 对象时可能会提供删除选项。"
+title: "删除选项"
+weight: 1
+auto_generated: true
+---
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:23]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+
+`import "k8s.io/apimachinery/pkg/apis/meta/v1"`
+
+<!--DeleteOptions may be provided when deleting an API object.-->
+删除 API 对象时可能会提供删除选项。
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:28]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+  APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+-->
+
+- **apiVersion** (string)
+
+  `APIVersion` 定义对象表示的版本化模式。
+  
+  服务器应将已识别的模式转换为最新的内部值，并可能拒绝无法识别的值。
+  
+  更多信息：https ://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:44]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+- **dryRun** ([]string)
+
+  When present, indicates that modifications should not be persisted. An invalid or unrecognized dryRun directive will result in an error response and no further processing of the request. Valid values are: - All: all dry run stages will be processed
+-->
+
+- **dryRun** ([]string)
+
+  该值如果存在，则表示不应保留修改。
+  
+  无效或无法识别的 `dryRun` 指令将导致错误响应并且不会进一步处理请求。有效值为： - `All`：处理所有试运行阶段（Dry Run Stages）
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:56]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+
+<!--
+- **kind** (string)
+
+  Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+-->
+
+- **kind** (string)
+
+  `Kind` 是一个字符串值，表示此对象代表的 REST 资源。
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:78]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+
+<!--
+- **orphanDependents** (boolean)
+
+  Deprecated: please use the PropagationPolicy, this field will be deprecated in 1.7. Should the dependent objects be orphaned. If true/false, the "orphan" finalizer will be added to/removed from the object's finalizers list. Either this field or PropagationPolicy may be set, but not both.
+-->
+
+- **orphanDependents** (boolean)
+
+  已弃用：该字段将在 1.7 中弃用，请使用 `PropagationPolicy` 字段。
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:92]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+  
+  表示依赖对象是否应该是孤儿。如果为真/假，“孤儿”终结器（Finalizer）将会被从对象终结器（Finalizer）列表中添加上或者移除掉。
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:94]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+    Specifies the target ResourceVersion
+
+  - **preconditions.uid** (string)
+
+    Specifies the target UID.
+-->
+  
+- **preconditions** (Preconditions)
+
+  先决条件必须在执行删除之前完成。如果不是，将返回 409（冲突）状态。
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:117]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+  
+  <a name="Preconditions"></a>
+  *执行操作（更新、删除等）之前必须完成先决条件。*
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:120]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+
+  - **preconditions.resourceVersion** (string)
+
+    指定目标资源版本（ResourceVersion）。
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:124]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+
+  - **preconditions.uid** (string)
+
+    指定目标 `UID`.
```
[the remark this answers is not in the document]
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:128]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md (hunk 1 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:138]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md (hunk 2 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:140]: 已修改

File: content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md (hunk 3 above)
Comment (twilight0620) [reply content/zh/docs/reference/kubernetes-api/common-definitions/delete-options.md:142]: 已修改

**Comment by twilight0620** [comment]
I have already modified all comments. Thanks for your review.
/assign @tengqm

**Comment by howieyuen** [comment]
tide/merge-method-squash

**Pull request merged**